examples/tests/gendata.py

```diff
@@ -18,7 +18,7 @@ def test_bsod(self):
             from PIL import Image
             from PIL import ImageDraw
         except ImportError:
-            return
+            self.cancel("PIL not available")
```

Comment:
I'd happily drop this test method altogether.

Reply:
Yeah, I don't love too... maybe some emotional attachment though :)

Anyway, just thought it was a lesser impact...